containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/n-install.sh

```diff
@@ -47,6 +47,9 @@ if [ "${n_install_result}" = "false" ]; then
   rm -f "${N_PREFIX}" && exit 6
 fi
 
+# prune orphaned Node.js versions
+log-stderr.sh n prune || true
```

Comment:
I wasn’t sure if including `log-stderr.sh` here was worth it given that this command isn’t really essential. Happy to remove it.

Reply:
It's fine to use `log-stderr.sh` here.